### DISCUSSION TITLE
How to find settings page after a Windows update / after turning off Explorer Patcher?

### DISCUSSION THREAD

**Discussion by hazymat** [valinet/ExplorerPatcher#906] [General]
Could you please update the installation instructions on this GitHub, to include how to find the settings page, when you have turned off the patcher OR after a Windows update?

For example I have just switched to Windows Insider developer mode, and when I right-click on the taskbar there is no option to find Explorer Patcher settings. Only Win11 Taskbar Settings. When I try to reinstall from the setup program left in C:\Program Files, it gives an a friendly error message, however none of the troubleshooting steps work (e.g. reboot PC, restart explorer.exe) and I can't reinstall it.

When I try to reinstall the software by re-downloading it, I get no error message, but instead Windows Explorer restarts and the settings dialog is not shown. I have tried this after a clean reboot as well.

Thanks

**Comment by Andre-10** [comment]
New experimental versions have EplorerPatcher "Properties" entry in Start menu.

For older (stable) version, you can start Task manager, and under file select "Run new task" then enter:

`rundll32.exe C:\Windows\dxgi.dll,ZZGUI`
